addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+<!-- assets -->
```

Comment:
@nim-odoo Fine to add assets views in stable? If so, would it be ok to add it in a assets.xml file instead?

Reply:
OK to add assets in stable. Indeed, seems to make sense to add it in assets.xml since it is used in the backend as well

https://github.com/odoo/odoo/blob/fe9d26fbca666d732e57ce5d5c6bd38adf1cea03/addons/website_hr_recruitment/models/hr_recruitment.py#L45